cdap-app-fabric/src/main/java/co/cask/cdap/lineage/field/FieldLineageInfo.java

```diff
@@ -67,16 +84,21 @@
    * @throws IllegalArgumentException if validation fails
    */
   public FieldLineageInfo(Collection<? extends Operation> operations) {
-    Set<String> operationNames = new HashSet<>();
     Set<String> allOrigins = new HashSet<>();
+    this.operationsMap = new HashMap<>();
+    this.writeOperations = new HashSet<>();
     boolean readExists = false;
     boolean writeExists = false;
```

Comment:
is this still needed? Isn't it equivalent to !writeOperations.isEmpty()

Reply:
its not now.. removed.